### PR TITLE
UI Touchups

### PR DESCRIPTION
**Pull request by HunterBarclay**
### Description
Getting the UI ready for users. Little things here and there I don't mind, but I want to get general issues that inhibit the usage of the application out of the way.

### Tasks
- [x] Fixed overflow on import panel
- [x] Removed most buttons from MainHUD
- [x] Moved debugging and testing related buttons to a new debug panel
- [x] Added missing icons to MainHUD, modals, and panels

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1739)

### CODE REVIEW

File: fission/src/ui/panels/DebugPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    World.PhysicsSystem.SetBodyPosition(ghostBody.GetID(), new JOLT.Vec3(2, 2, 2))
+}
+
+const DebugPanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
+    const { openPanel } = usePanelControlContext()
+
+    return (
+        <Panel
+            openLocation="center"
+            name={"The Poker"}
```
Comment (Dhruv-0-Arora) [inline fission/src/ui/panels/DebugPanel.tsx]:
```suggestion
            name={"Debug Panel"}
```

**Review by Dhruv-0-Arora — CHANGES_REQUESTED** [review]

**Comment by LucaHaverty** [comment]
Do we need multibot on the main hud anymore?

**Review by Dhruv-0-Arora — CHANGES_REQUESTED** [review]
Could you resolve merge conflicts rq 😄

**Review by LucaHaverty — APPROVED** [review]
Looks great! Just a quick merge and It'll be perfect.

**Review by Dhruv-0-Arora — APPROVED** [review]
Looks good 👍

**Pull request merged**